src/pygeoprocessing/testing/sampledata.py

```diff
@@ -113,6 +113,11 @@ def projection_wkt(epsg_id):
     origin=(443723.127327877911739, 4956546.905980412848294),
     pixel_size=lambda x: (x, -1. * x)
 )
+SRS_WGS84 = ReferenceData(
+    projection=projection_wkt(4326),
+    origin=(-123.71107369348861, 44.76009901797307),
+    pixel_size=lambda x: (x, -1. * x)
+)
```

Comment:
Is this used anywhere? I tried to "ctrl+f" it and nothing turned up. If not, can you remove it?

Reply:
Good point! I think I added it when I was creating earlier tests before we talked about having the `testing` module go away. And then I refactored those tests!

Reply:
Removed in [03b39f](https://github.com/natcap/pygeoprocessing/pull/31/commits/03b39fec8d4b72c9b1315082b19c4c171f14c79c)